registration-processor/init/registration-processor-registration-status-service/src/main/java/io/mosip/registration/processor/status/api/controller/RegistrationExternalStatusController.java

```diff
@@ -82,7 +72,7 @@
 	 */
 	@PreAuthorize("hasAnyRole('REGISTRATION_ADMIN', 'REGISTRATION_OFFICER', 'REGISTRATION_SUPERVISOR','RESIDENT')")
 	@PostMapping(path = "/externalstatus/search", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-	@ApiOperation(value = "Get the registration external status", response = RegistrationExternalStatusCode.class)
+	@Operation(summary = "Get the registration external status", description = "registration external status", tags = { "External Registration Status" })
```

Comment:
For all APIs have the summary and description as same for now

Reply:
Yes, I changed all summaries and descriptions.